types/nightmare/index.d.ts

```diff
@@ -96,6 +96,7 @@ declare class Nightmare {
     removeListener(event: 'error', cb: (msg: string, trace?: Nightmare.IStackTrace[]) => void): Nightmare;
     removeListener(event: 'timeout', cb: (msg: string) => void): Nightmare;
     screenshot(path: string): Nightmare;
+    screenshot(path: string, clip: Object): Nightmare;
```

Comment:
Could you add an explicit type for the shape of `clip`? Either a declared one, or a type literal like `{ x: number, y: ... }`

Also, [according to the source](https://github.com/segmentio/nightmare/blob/a835558067c0dd72b5b10c14613d9071aef39d13/lib/actions.js#L586-L602), it looks like the appropriate overloads here would be

```ts
screenshot(path: string, done?: (err: any) => void): Nightmare;
screenshot(clip: { x: number, y: number, width: number, height: number }, done?: (err: any) => void): Nightmare;
screenshot(path: string, clip?: { x: number, y: number, width: number, height: number }, done: (err: any) => void): Nightmare;
```